siddhi-extensions/nlp/src/main/java/org/wso2/siddhi/gpl/extension/nlp/NameEntityTypeViaDictionaryStreamProcessor.java

```diff
@@ -45,7 +45,7 @@
     private Dictionary dictionary;
 
     @Override
-    protected List<Attribute> init(AbstractDefinition abstractDefinition, ExpressionExecutor[] attributeExpressionExecutors, ExecutionPlanContext executionPlanContext, boolean outputExpectsExpiredEvents) {
+    protected List<Attribute> init(AbstractDefinition abstractDefinition, ExpressionExecutor[] attributeExpressionExecutors, ExecutionPlanContext executionPlanContext) {
         if (logger.isDebugEnabled()) {
             logger.debug("Initializing Query ...");
```

Comment:
Always add context information to debug queries. In this case content of the query if possible